src/api/env.production

```diff
@@ -0,0 +1,22 @@
+################################################################################
+# Environment Variables for Production
+#
+# Copy this file to `.env` for use by docker-compose.
+################################################################################
+
+COMPOSE_PROJECT_NAME=telescope_api
+
+# Compose files to use together on production. NOTE: we specify separator below
+# so it will work on Windows and Unix, see
+# https://docs.docker.com/compose/reference/envvars/#compose_file
+COMPOSE_PATH_SEPARATOR=;
+COMPOSE_FILE=docker-compose.yml;production.yml
+
+# The host where all the microservices run (e.g., http://api.telescope.localhost)
+API_HOST=api.telescope.cdot.systems
```

Comment:
Since both `env.staging` and `env.production` are identical except for `API_HOST`, we could only have `env.production` and modify the value of `API_HOST` doing something similar to what we do with the deployment server, something like [this](https://github.com/Seneca-CDOT/telescope/blob/cc8151f3c6e5435bed4c2c6237aa4063d5c0f636/tools/autodeployment/deploy.sh#L30) or [this](https://github.com/Seneca-CDOT/telescope/blob/cc8151f3c6e5435bed4c2c6237aa4063d5c0f636/tools/autodeployment/deploy.sh#L42)

Reply:
I wondered about that, too. I felt like having a second file was less hacky than scripts that switch out strings, but I'm happy to do whatever.

Reply:
Though I'd argue using /sed is the cleanest method, we can't assume that ALL are working in a bash environment. I'd opt for two files which removes technical requirements. It's simpler.

Reply:
This would only affect `env.staging` and `env.production`, which are used in production deployment (presumably, production deployments will always take place in bash environments), so it's safe for local development.